api/swagger-spec/oapi-v1.json

```diff
@@ -17700,6 +17700,10 @@
      "customStrategy": {
       "$ref": "v1.CustomBuildStrategy",
       "description": "CustomStrategy holds the parameters to the Custom build strategy"
+     },
+     "jenkinsPipelineStrategy": {
```

Comment:
@jwforres I was already playing with this in console and there are some gotchas here:

1) I don't think 'start build' button has any meaning here, @bparees @jimmidyson? I tried to click on that button and the build will stay in 'New' forever ;-)
2) We have to show link to Jenkins in the build config overview page
3) I'm not sure how are we going to stream the logs from Jenkins (if we are going to do that) to console
4) I guess visualization of the pipeline will be important here